### DISCUSSION TITLE
Missing string data

### DISCUSSION THREAD

**Discussion by pw1031** [jasonacox/Powerwall-Dashboard#207] [General]
Hi,

I have Powerwall+ and an extra inverter (3.8 version), my solar system has 4 strings in total, 2 per inverter.

Will I be able to monitor every string individually with this setup? and if so, how do I do it? because I only see two strings (C and D) reporting data in my Dashboard. 

Appreciate any help. 

Thanks!

**Comment by jasonacox** [comment]
You will only be able to see the strings that are part of the inverter built in to the Powerwall+.  There is currently no way known to get the string data from the external inverters, unfortunately. If anyone discovers a way, please do post it.
